libs/engine/src/app/components/context-picker.tsx

```diff
@@ -16,6 +16,8 @@ export const ContextPicker: FC = () => {
   return (
     <ContextTree>
       {({ context }) => {
+        if (context.contextType === 'root' || context.contextType === 'website') return null
```

Comment:
add todo and explain there why it's needed, looks like tech debt

add namespace "engine" for the website context
maybe it makes sense to add some checks that root context is actually root, eg:
```ts
function isTopContext(context: IContextNode): boolean {
    return context.contextType === 'website' && context.namespace === 'engine';
}

if (context.contextType === 'root' && isTopContext(context.parent) || isTopContext(context)) return null;
```

Reply:
👌